packages/cli/src/Init.ts

```diff
@@ -344,26 +352,49 @@ export class Init implements Command {
         console.log(`Successfully authenticated as ${bold(authenticationResult.email)}.`)
       }
 
+      if (args['--generate']) {
```

Comment:
What is the intented usage of `--generate` flag? I could not see it being used by the MCP command.

Reply:
I am going to rename it to `--prompt` with an alias for `--vibe` as we discussed. This flag will use AI to generate a schema based on the provided prompt:

```
npx prisma init --prompt "A todo app specifically for tracking birds"
```